spark/src/test/scala/org/apache/spark/sql/delta/DeletionVectorsTestUtils.scala

```diff
@@ -110,6 +110,11 @@ trait DeletionVectorsTestUtils extends QueryTest with SharedSparkSession with De
     super.withTempDir(prefix = "s p a r k %2a")(f)
   }
 
+  /** Helper that checks that log contains DVs */
+  def checkDVsExist(targetLog: DeltaLog): Boolean = {
+    getFilesWithDeletionVectors(targetLog).nonEmpty
```

Comment:
Adding a helper method without actual usages doesn't make sense for me. Also, why not check the presence in caller side?